entity-framework/core/managing-schemas/migrations/projects.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: Migrations with Multiple Projects - EF Core
+title: Using a Separate Project - EF Core
 author: bricelam
 ms.author: bricelam
 ms.date: 10/30/2017
 uid: core/managing-schemas/migrations/projects
 ---
-Using a Separate Project
-========================
-You may want to store your migrations in a different assembly than the one containing your `DbContext`. You can also
-use this strategy to maintain multiple sets of migrations, for example, one for development and another for
-release-to-release upgrades.
+
+# Using a Separate Project
```

Comment:
@bricelam - H1 and title should match. What is the right one to use here?

Reply:
Should probably be *Using a Separate Migrations Project*.